.github/workflows/pull-request.yml

```diff
@@ -28,6 +28,12 @@ jobs:
       - name: Run Linter
         run: |
           swiftlint --strict
+
+      - name: Validate Pull Request Name
+        id: versioning
+        uses: Oliver-Binns/Versioning@1.0.0
```

Comment:
This should be :
uses: Oliver-Binns/Versioning@COMMITSHA # 1.0.0

Reply:
https://gds-way.digital.cabinet-office.gov.uk/standards/source-code/use-github.html#using-github-actions-and-workflows

Reply:
We have created a ticket to update this! The change will be made for these repos `mobile-ios-networking`, `mobile-ios-logging`, `mobile-ios-coordination`, `mobile-ios-authentication` and `mobile-ios-common` :)